Explorer/Assets/DCL/Multiplayer/Movement/Systems/PlayerMovementNetSendSystem.cs

```diff
@@ -91,28 +101,54 @@ private static void UpdateMessagePerSecondTimer(float t, ref PlayerMovementNetwo
             }
         }
 
-        private void SendMessage(ref PlayerMovementNetworkComponent playerMovement, in IAvatarView view, in CharacterAnimationComponent animation, in StunComponent playerStunComponent, in MovementInputComponent movement)
+        private void SendMessage(ref PlayerMovementNetworkComponent playerMovement, in CharacterAnimationComponent animation, in StunComponent playerStunComponent, in MovementInputComponent movement)
         {
             playerMovement.MessagesSentInSec++;
 
+            // We use this calculation instead of Character.velocity because, Character.velocity is 0 in some cases (moving platform)
+            float dist = (playerMovement.Character.transform.position - playerMovement.LastSentMessage.position).magnitude;
+            float speed = dist / (UnityEngine.Time.unscaledTime - playerMovement.LastSentMessage.timestamp);
+
+            int tier;
+
+            if (speed < settings.VelocityTiers[0]) tier = 0;
```

Comment:
you can replace it with a loop according to the number of elements in `settings.VelocityTears`